src/components/app/ProfileSharing.js

```diff
@@ -142,9 +146,10 @@ class ProfileSharingCompositeButton extends PureComponent {
     return shortenUrl(this.state.fullUrl)
       .then(shortUrl => {
         this.setState({ shortUrl });
-        if (this._permalinkTextField) {
-          this._permalinkTextField.focus();
-          this._permalinkTextField.select();
+        const textField = this._permalinkTextField;
+        if (textField) {
```

Comment:
This needs to be in a separate const because otherwise Flow thinks the call to `focus` may nullify the property.

Reply:
Makes sense. I think this is a fairly common pattern to me at this point.